test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -699,6 +904,93 @@ func BenchmarkPerfScheduling(b *testing.B) {
 	}
 }
 
+var testSchedulingLabelFilter = flag.String("test-scheduling-label-filter", "unittest", "comma-separated list of labels which a testcase must have (no prefix or +) or must not have (-)")
```

Comment:
> I don't think these are unit tests. 

Let's replace "unittest" with "functionality" or "integration-tests".

> But we have dedicated integration tests. Why add this?

Two reasons:
- It ensures that the code going into scheduler_perf gets tested in a pre-merge Prow job. Previously, linting only ensured that it compiles, but nothing actually ran the code.
- Last but not least, it reduces the amount of test code that needs to be written for new features like DRA. The same test can cover correctness and performance checks, depending on the configuration. Simulating special scenarios may still need separate tests, but the basic "does a pod get scheduled with dynamic resource claim" is covered well here already.

I was about to add that race detection can be enabled when performance isn't relevant (there were bugs in scheduler_perf found by that), but "pull-kubernetes-integration" currently doesn't do that. I'm checking in #sig-testing whether it could get enabled.